models/dex/dex_aggregator_trades.sql

```diff
@@ -31,6 +31,7 @@ spells with issues, to be excluded in short term:
     ,ref('dodo_aggregator_trades')
     ,ref('zeroex_trades')
     ,ref('kyberswap_aggregator_trades')
+    ,ref('tokenlon_trades')
```

Comment:
both [this](https://github.com/duneanalytics/spellbook/blob/main/models/tokenlon/ethereum/tokenlon_ethereum_trades.sql) spell and [this](https://github.com/duneanalytics/spellbook/blob/main/models/tokenlon/tokenlon_trades.sql) spell are not pulling `block_month` through, so it's failing when trying to union into the sector spell. can you look to pull that column through and try tests again?

Reply:
OK, I'll try, thank you